demo/java/demotest

```diff
@@ -10,7 +10,6 @@ RunDemo()
     java -jar build/libs/astronomy-demo-0.0.1.jar $*
 }
 
-echo "Build astronomy java demo"
```

Comment:
I added this in the previous PR and felt better to clean it up now that it has lost its purpose